src/patterns/definitions.owl

```diff
@@ -7,8 +7,142 @@ Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)
+Declaration(AnnotationProperty(<http://purl.org/dc/terms/contributor>))
+Declaration(AnnotationProperty(<http://www.geneontology.org/formats/oboInOwl#hasDbXref>))
+
+
+
+############################
+#   Classes
+############################
+
+# Class: <http://purl.obolibrary.org/obo/CL_4033000> (blood vessel endothelial cell of venule of lymph node)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "DOI:10.3389/fimmu.2017.00045"^^xsd:string) Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:28217126"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/CL_4033000> "A(n) blood vessel endothelial cell that is part of a(n) venule of lymph node."^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> <http://purl.obolibrary.org/obo/CL_4033000> <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/CL_4033000> "blood vessel endothelial cell of venule of lymph node"^^xsd:string)
+EquivalentClasses(<http://purl.obolibrary.org/obo/CL_4033000> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/CL_0000071> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> <http://purl.obolibrary.org/obo/UBERON_8410041>)))
+
+# Class: <http://purl.obolibrary.org/obo/CL_4033001> (blood vessel endothelial cell of arteriole of lymph node)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "DOI:10.3389/fimmu.2017.00045"^^xsd:string) Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:28217126"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/CL_4033001> "A(n) blood vessel endothelial cell that is part of a(n) arteriole of lymph node."^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> <http://purl.obolibrary.org/obo/CL_4033001> <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/CL_4033001> "blood vessel endothelial cell of arteriole of lymph node"^^xsd:string)
+EquivalentClasses(<http://purl.obolibrary.org/obo/CL_4033001> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/CL_0000071> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> <http://purl.obolibrary.org/obo/UBERON_8410042>)))
+
+# Class: <http://purl.obolibrary.org/obo/CL_4033002> (neuroendocrine cell of epithelium of crypt of Lieberkuhn)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "DOI:10.1159/000501521"^^xsd:string) Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMCID:PMC6888884"^^xsd:string) Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:31390635"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/CL_4033002> "A(n) neuroendocrine cell that is part of a(n) epithelium of crypt of Lieberkuhn."^^xsd:string)
```

Comment:
Of note, the reference describes neuroendocrine cells in the "deep mucosal layer along the muscularis mucosa" in a subset of non-pathology specimens.

Reply:
The same reference appears as three separate dbxrefs with different identifiers. My understanding is that PubMed Identifiers (PMID) are [preferred](https://oboacademy.github.io/obook/lesson/contributing-to-obo-ontologies/#adding-annotations) and that different dbxrefs pointing to the same resource is not standard.

Reply:
Well spotted.  Problem is the source - ASCT+B @aleixpuigb  - we should check whether different identifiers are duplicates before adding.

Reply:
done